File: SISTEMA_DE_VOTACAO/sistemaDeVotacao.java
import java.util.*;

public class sistemaDeVotacao {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String[] opcoes = {"Opção 1", "Opção 2", "Opção 3"};
        int[] votos = new int[opcoes.length];
        boolean votacaoAtiva = true;

        System.out.println("=== SISTEMA DE VOTAÇÃO ===");
        System.out.println("As opções são:");
        for (int i = 0; i < opcoes.length; i++) {
            System.out.println((i + 1) + " - " + opcoes[i]);
        }

        while (votacaoAtiva) {
            System.out.print("\nEscolha uma opção (1-" + opcoes.length + ") ou 0 para encerrar: ");

            if (scanner.hasNextInt()) {
                int escolha = scanner.nextInt();

                if (escolha == 0) {
                    votacaoAtiva = false;
                } else if (escolha >= 1 && escolha <= opcoes.length) {
                    votos[escolha - 1]++;
                    System.out.println("Você votou na " + opcoes[escolha - 1]);
                } else {
                    System.out.println("Opção inválida. Tente novamente.");
                }
            } else {
                System.out.println("Entrada inválida. Digite apenas números.");
                scanner.next(); // descarta entrada inválida
            }
        }

        System.out.println("\n=== RESULTADOS FINAIS ===");
        for (int i = 0; i < opcoes.length; i++) {
            System.out.println(opcoes[i] + ": " + votos[i] + " voto(s)");
        }

        scanner.close();
    }
}

File: SISTEMA_DE_VOTACAO/sistemaDeVotacao.py
opcoes = ['Opção 1', 'Opção 2', 'Opção 3']
votos = [0, 0, 0]
votacaoAtiva = True
print("=== SISTEMA DE VOTAÇÃO ===")
print('As opções são: ')
for i  in range(len(opcoes)):
    print(f'{i+ 1} - {opcoes[i]}')
    
while votacaoAtiva:
    escolha = input(f'\nEscolha uma opção (1-{len(opcoes)}) ou 0 pra encerrar: ')
    if escolha.isdigit():
        escolha_int = int(escolha)
        if escolha_int == 0:
            votacaoAtiva = False
        elif escolha_int == 1:
            votos[0] += 1
            print(f'Você votou na {opcoes[0]}')
        elif escolha_int == 2:
            votos[1] += 1
            print(f'Você votou na {opcoes[1]}')
        elif escolha_int == 3:
            votos[2] += 1
            print(f'Você votou na {opcoes[2]}')
    else:
        print('Entrada inválida, digite apenas números.')
            
print(f'\n=== RESULTADOS FINAIS ===')
for i  in range(len(opcoes)):
    print(f'{opcoes[i]}: {votos[i]}')

